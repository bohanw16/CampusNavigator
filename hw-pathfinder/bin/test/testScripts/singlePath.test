# test two nodes with a single path

CreateGraph g

AddNode g n1
AddNode g n2

AddEdge g n1 n2 1.111

FindPath g n1 n2