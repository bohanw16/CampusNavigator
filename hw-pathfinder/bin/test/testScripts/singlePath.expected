# test two nodes with a single path

created graph g

added node n1 to g
added node n2 to g

added edge 1.111 from n1 to n2 in g

path from n1 to n2:
n1 to n2 with weight 1.111
total cost: 1.111